src/interceptors/on-response-refresh-token.ts

```diff
@@ -0,0 +1,49 @@
+import axios, { AxiosError, InternalAxiosRequestConfig } from 'axios';
+import { StatusCodes } from 'http-status-codes';
```

Comment:
This data can be taken from axios as well

```suggestion
import { HttpStatusCode } from 'axios';
```